test/testcafe/spec/IdentifyWithPassword_spec.js

```diff
@@ -83,7 +83,7 @@ test.requestHooks(identifyWithPasswordMock)('should show customized error if req
   await identityPage.waitForErrorBox();
 
   await t.expect(identityPage.hasIdentifierErrorMessage()).eql(true);
-  await t.expect(identityPage.getIdentifierErrorMessage()).eql('Username is required!');
+  await t.expect(identityPage.getIdentifierErrorMessage()).match(/Username is required!/);
```

Comment:
We have to use `.match(/..../)` because any optional Odyssey fields will bring in the "Optional" string as text content when trying to use a selector